manifests/0000_50_cluster_monitoring_operator_02-role.yaml

```diff
@@ -145,6 +147,25 @@ rules:
   - namespaces
   verbs:
   - get
+- apiGroups:
```

Comment:
Why do we need this?

Reply:
I'm not sure, it was automatically generated by the script...

cc @s-urbaniak @paulfantom